src/Sulu/Bundle/ContentBundle/Resources/public/js/validation/types/resourceLocator.js

```diff
@@ -40,7 +40,7 @@ define([
                     var val = this.getValue(),
                         part = App.dom.data($el, 'part');
 
-                    return part.length > 0 && val !== '/' && validator.exec(val);
+                    return part.length > 0 && val !== '/' && validator.test(val);
```

Comment:
Did you agree with somebody that simply adjusting the regex is enough? /cc @chirimoya